src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/ios/CarinaDescriptionPage.java

```diff
@@ -20,6 +20,26 @@ public CarinaDescriptionPage(WebDriver driver) {
 		super(driver);
 	}
 
+	@Override
+	public WebViewBase navitateToWebview() {
+		return null;
+	}
+
+	@Override
+	public ChartsBase navigateToCharts() {
+		return null;
+	}
+
+	@Override
+	public MapsBase navigateToMap() {
+		return null;
+	}
+
+	@Override
+	public UIElementsBase navigateToUIElements() {
```

Comment:
- Same as describe above.
- Please rename according to Project Naming Convenience.